lib/config/recommended.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     'img-alt-attributes': true,
     'style-concatenation': true,
     'deprecated-inline-view-helper': true,
-    'unused-block-params': true
+    'unused-block-params': true,
+    'inline-styles': true
```

Comment:
We have to wait on changing the recommended set for 0.7.0 (as it is a "breaking" change, since it could fail apps that were previously passing using recommended).

Can you back out this change?